posthog/models/dashboard_item.py

```diff
@@ -45,6 +46,9 @@ class DashboardItem(models.Model):
     short_id: models.CharField = models.CharField(
         max_length=12, blank=True, default=generate_short_id,
     )  # Unique ID per team for easy sharing and short links
+    updated_at: models.DateTimeField = models.DateTimeField(auto_now=True)
+    tags: ArrayField = ArrayField(models.CharField(max_length=32), blank=True, default=list)
+    favorited: models.BooleanField = models.BooleanField(default=False)
```

Comment:
Pinning is super specific to the dashboard launch menu. Insights will probably have a much higher volume of records, so I went with "favorites" here. It's really more of a reserved tag than a "stick this to the top of the list" action.

Reply:
I just meant for the internal attribute name, not necessarily what we do user-facing

Reply:
Do we still want to call it `favorited` on the front then? I think it's better to refer to things the same way, so if we change it to `pinned`, we should be changing the stars to pins and the "favorited" tab to "pinned", what do you think? @clarkus

Reply:
I also feel like the functionality of saved / pinned is quite similar, and saved / favorited is slightly more distinguishable?

Reply:
I mean we can just refer to it as "favorited" in the front but then we'd have to leave a comment and remember that it's called something else in the backend and for dashboards pinning is actually a pin. I think we should keep our mental models of these things as clear/concise as possible? Maybe pinning dashboards could be "favorites" instead too?

Reply:
Where are we landing on this point? I'm remaking the migrations because of merge conflicts, so I can change to `pinning` if we feel very strongly about it, otherwise I'm going to leave `favoriting` as favoriting and `pinning` as pinning, since they have different connotations